contracts/core/modules/CTokenExchangeIssuanceModule.sol

```diff
@@ -289,13 +290,16 @@ contract CTokenExchangeIssuanceModule is
         uint256[] memory baseSetUnits = baseSet.getUnits();
         uint256 baseSetNaturalUnit = baseSet.naturalUnit();
 
+        // Calculate the number of natural units required and round down to nearest natural unit
```

Comment:
Maybe add note that we are guaranteed this is an integer because we check that `quantity.mod(naturalUnit) == 0`